packages/@aws-cdk/aws-batch/test/integ.batch.expected.json

```diff
@@ -0,0 +1,1874 @@
+{
```

Comment:
I'm 90% sure that this file is not needed and shouldn't be a part of this PR. `expected.json` files are a relic of our legacy integration test system which are now replaced by the `batch.integ.snapshot` folder. I am not sure why/how this file was created, but I suspect it has something to do with the initial PR (from March) coming before we introduced the new integ system.

Can you please delete this file and then run `yarn integ` in your batch folder again? This file should not get created and that will verify what I'm saying is true.

Reply:
You are correct. I delete expected.json and re-ran `yarn integ` and it passed the test!